client/src/components/common/ScrollSpy/ScrollSpy.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import { Scrollspy } from '@makotot/ghostui';
```

Comment:
Be more skeptical about importing libraries with low weekly download count, these libraries often not maintained well.